plugins/music.js

```diff
@@ -5,11 +5,23 @@ module.exports = function(pluto) {
 
     var musicModule = {
         lastPlaying: null,
+        progress: null,
         queue: [],
         paused: false,
         lastMessage: null
     };
 
+    pluto.addListener("player::progress", function(data) {
```

Comment:
its fine to have this endpoint. maybe for efficiency's sake we can infer the time after getting the start event. So update the time on the front end. after start and poll less often(every 10 seconds) to bring the time up to parity.